data/systemd/snap.mount.in

```diff
@@ -0,0 +1,15 @@
+# trusty needs this to make /snap rshared
+[Unit]
+Description=Mount unit for @SNAP_MOUNT_DIR@
```

Comment:
The description is shown by Plymouth so perhaps we can say something more specific
```suggestion
Description=Special mount event propagation for @SNAP_MOUNT_DIR@
```